.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][RHIDP#<jira-issue-id>]: <short-description-of-the-pr> --->
+
+<!--- If your changes apply to the latest release and/or in-development version of RHDH, open your PR against the `main` branch, then ensure that your PR is cherrypicked to any appropriate released branches. --->
+
+**Version(s):**
+<!--- Specify the version(s) of RHDH your PR applies to. -->
+
+**Issue:**
+<!--- Add a link to the Bugzilla, Jira, or GitHub issue. --->
+
+**Link to docs preview:**
+<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->
+
+**Reviews:**
+- [ ] SME: @ mention assignee
+- [ ] QE: @ mention assignee
+<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->
+- [ ] Docs peer review: @ mention assignee
+<!--- Docs peer review approval is required for ALL PRs. --->
+
+**Additional information:**
+<!--- Optional: Include additional context or expand the description here.--->
+
+<!--- After you open your PR, ask for review from the RHDH docs team via Slack. --->
```

Comment:
```suggestion
<!--- After you open your PR, request reviews all required stakeholders via Slack GitHub, or Jira. --->
```
Eh? It would be nice for ANY contributor to take responsibility for meeting all review requirements instead of defaulting SME/QE reviewer-chasing tasks to the Docs team. And how they do it is up to them. WDYT?